app/publishing/calculation/get-unpublished.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../data')
+
+const getUnpublished = async (transaction) => {
+  const calculations = await db.calculation.findAll({
+    where: { published: null },
+    include: [{ model: db.funding, as: 'fundings', attributes: ['calculationId', 'fundingCode', 'areaClaimed', 'rate'] }],
+    nest: true,
+    transaction
+  })
+  return calculations.map(x => x.get({ plain: true }))
```

Comment:
Limitation of Sequelize that cannot nest one to many results in raw query.  

Instead of:
```
[{
  fundings: [{
    id: 1
  }, {
    id: 2
  }]
}]
```
it would return
```
[{
  fundings: {
    id: 1
  }
}, {
  fundings: {
    id: 2
  }
}]
```

Therefore extracting plain objects from sequelize objects to achieve same result